apps/examples/st_things/light/sample_light_things.c

```diff
@@ -52,6 +53,15 @@ static void gpio_callback_event(void *user_data)
 }
 #endif
 
+#ifdef CONFIG_LIGHT_SWITCH_BUTTON
+static void gpio_callback_event_trigger_light_switch(void *user_data)
+{
+	printf("gpio call back event for notify switch resource observers manually!!\n");
```

Comment:
Change the text to: gpio_callback_event to notify switch resource's observers manually